src/test/java/com/softserve/edu/greencity/ui/pages/tipstricks/TipsTricksPage.java

```diff
@@ -91,7 +77,8 @@ public boolean isDisplayedStartHabitTop() {
     // ButtonCenter 'Start forming a habit'
 
     public WebElement getStartHabitCenter() {
-        return startHabitCenter;
+        return startHabitCenter = driver
+                .findElement(By.cssSelector(".stat-row-content.ng-star-inserted:nth-child(2)> div> button"));
```

Comment:
need to rewrite this selector

Reply:
Change to:
startHabitCenter = driver
                .findElement(By.xpath(".stat-row-content:nth-child(2) .button"));